k8_cortx_cloud/charts/cortx/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+appVersion: 2.0.0-725
```

Comment:
Most Charts set this to the same as the image versions, so I'm matching this with what's in solution.example.yaml.

Reply:
The only question I have here is that would you expect we only update the appVersion here whenever we cut a release for cortx-k8s? IE it's not perpetually always generating a new chart version whenever a new container version is available?

Reply:
I think we should bump the appVersion anytime we change the default image version. Basically it's the equivalent of updating solution.example.yaml. If we cut a new release but no images have changed, then no reason to change appVersion. However, if we do update the appVersion, then that also means a chart version bump.

Reply:
> IE it's not perpetually always generating a new chart version whenever a new container version is available?

Right, this field is updated manually to whichever version we've qualified for a new release. Of course, it could be automated by a release action, but it's not meant to be updated for every upstream build.